### ISSUE TITLE
Incorrect number of FORMAT/PL values in .g.vcf

### ISSUE THREAD

**Issue by dennishendriksen** [HKU-BAL/Clair3#191]
I'm running into an issue where Clair3 produces invalid `.g.vcf` output resulting in errors when used with downstream tooling, e.g. BCFtools:
```
Incorrect number of FORMAT/PL values at chr2:14862711, cannot merge. The tag is defined as Number=G, but found
  4 values and 3 alleles. See also http://samtools.github.io/bcftools/howtos/FAQ.html#incorrect-nfields
```

Example output (unfortunately I can't share the input due to its sensitive nature):

`pileup.vcf.gz`:
```
chr2    14862711        .       C       .       7.03    RefCall P       GT:GQ:DP:AD:AF:PL       0/0:7:2:1:0.5000:2970
```
`full_alignment.vcf.gz`:
```
chr2    14862711        .       C       N       7.97    PASS    F       GT:GQ:DP:AD:AF:PL       0/1:7:3:1,1:0.3333:2970
```
`merge_output.vcf.gz`:
```
chr2    14862711        .       C       N       7.97    PASS    F       GT:GQ:DP:AD:AF:PL       0/1:7:3:1,1:0.3333:2970
```
`output.g.vcf.gz`:
```
chr2    14862711        .       C       N,<NON_REF>     7.97    PASS    F       GT:GQ:DP:AD:AF:PL       0/1:7:3:1,1,0:0.3333:2970,990,990,990
```

I think the error is in the `PL` computation in case the alternate allele value is `N`, caused by https://github.com/HKU-BAL/Clair3/blob/v1.0.0/clair3/CallVariants.py#L1395? Since the VCF field `PL` is of type `G` the field still must have one value for each possible genotype according to the VCF specification.

Possibly the following was intended in `full_alignment.vcf.gz`?
```
chr2    14862711        .       C       N       7.97    PASS    F       GT:GQ:DP:AD:AF:PL       0/1:7:3:1,1:0.3333:990,990,990
```

**Comment by aquaskyline** [comment]
To be fixed in the next release.

**Comment by zhengzhenxian** [comment]
@dennishendriksen Fixed in v1.0.1, pls kindly try, thanks!
